src/core/src/sim/md/controls.rs

```diff
@@ -181,6 +181,26 @@ impl Control for RemoveRotation {
     }
 }
 
+
+/******************************************************************************/
+/// Rewrap all molecules inside the unit cell
```

Comment:
Maybe we can add that not all atoms of a molecule will lie within the unit cell.
```rust
/// Rewrap all molecules' centers of mass to lie within the unit cell. Atoms of a molecule may lie outside of the cell.